### PR TITLE
Add Gitignore to ignore compilation tool files

### PR DESCRIPTION
**Pull request by matthewei**
Add Gitignore to ignore compilation tool files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,6 @@ go.work
 
 # Generated binaries
 bin/
+
+.idea
+.vscode/
```
Comment (tuminoid) [inline .gitignore:24]: Missing linefeed at the end.

Comment (matthewei) [reply .gitignore:24]: done

Comment (matthewei) [reply .gitignore]: I have add it.

**Review by tuminoid — COMMENTED** [review]
/ok-to-test
/lgtm

**Comment by matthewei** [comment]
@dtantsur Can we merge it?

**Comment by dtantsur** [comment]
/approve

Thanks!

**Pull request merged**